arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1033,10 +1033,6 @@
 				<string>GraphicsOverlayDictionaryRenderer3D</string>
 				<key>descriptionText</key>
 				<string>Create graphics using a local mil2525d style file and an XML file with key-value pairs for each graphic, and display the military symbols in 3D.</string>
-				<key>dependency</key>
```

Comment:
Because we load from portal instead of local file, I removed the dependency for the sample. However, the `mil2525d` file is also used by `FeatureLayerDictionaryRendererViewController`, so I don't remove it from the project. Maybe one day it can also be updated to using the portal item.